.github/workflows/e2e.yml

```diff
@@ -0,0 +1,67 @@
+name: End to End Tests
+on: 
+  workflow_dispatch:
+    inputs:
+      automationBranch:
+        description: 'Set the branch to use for automation tests'
+        required: false
+        default: 'develop'
+        type: string
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        os: [macos-latest]
+        vscodeVersion:
+          - stable
+    runs-on: ${{ matrix.os }}
+    steps:
+        - name: Checkout
+          uses: actions/checkout@v2
+          with:
+            path: ./salesforcedx-vscode
+            ref: ${{ github.event.ref }}
+        - uses: actions/setup-node@v3
+          with:
+            node-version-file: 'salesforcedx-vscode/.nvmrc' 
+            cache: npm
+            cache-dependency-path: |
+              salesforcedx-vscode/package-lock.json
+              salesforcedx-vscode-automation-tests/package-lock.json
+        - name: Clone automation tests
+          uses: actions/checkout@v2
+          with:
+            repository: forcedotcom/salesforcedx-vscode-automation-tests
+            path: salesforcedx-vscode-automation-tests
+            ref: ${{ inputs.automationBranch }}
```

Comment:
Fun fact - this used to be more verbose, as seen in our other workflows. ${{ github.event.inputs.automationBranch }} but according to documentation (and confirmed with running here) we can now omit the first two keys: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-of-onworkflow_callinputs